docs/docs/dev-guide-new-shield.md

```diff
@@ -302,3 +303,36 @@ and then flash with:
 ```
 west flash
 ```
+
+## Updating `build.yml`
+
+Before publishing your shield to the public via a PR, navigate to `build.yml` found in `.github/workflows` and add your shield to the appropriate list. An example edit to `build.yml` is shown below.
+
+```
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    name: Build Test
+    strategy:
+      matrix:
+        board: [proton_c, nice_nano, bluemicro840_v1, nrfmicro_13]
+        shield:
+          - corne_left
+          - corne_right
+          - kyria_left
+          - kyria_right
+          - lily58_left
+          - lily58_right
+          - iris_left
+          - iris_right
+          - romac
+		  - <MY_BOARD>
+		  - <MY_SPLIT_BOARD_left>
```

Comment:
Missed this the first go round, looks like some indent/whitespace inconsistency here.

Reply:
Just fixed the indents. Hopefully, that should be it.